web/src/modules/public/markets/sagas/marketPriceSaga.ts

```diff
@@ -0,0 +1,29 @@
+import { call, put } from 'redux-saga/effects';
+import { sendError } from '../../../';
+import { API, RequestOptions } from '../../../../api';
+import { getOrderAPI, buildQueryString } from '../../../../helpers';
+import {
+    marketPriceData,
+    marketPriceError,
+    MarketPriceFetch,
+} from '../actions';
+
+const marketPriceRequestOptions: RequestOptions = {
+    apiVersion: 'finex',
+};
+
+export function* marketPriceSaga(action: MarketPriceFetch) {
+    try {
+        const payload = buildQueryString(action.payload);
+        const price = yield call(API.get(marketPriceRequestOptions), `/public/qe/prices?${payload}`);
```

Comment:
Wrong query build. If there will be no payload - request URL will looks like `/public/qe/prices?`. Please, handle this.